### PR TITLE
Add templates to show talk types and tracks and their descriptions.

### PR DESCRIPTION
**Pull request by drnlm**
So we can link to them elsewhere, such as a "how to submit a talk" page or the like.

### CODE REVIEW

**Comment by hodgestar** [comment]
Woot. Given that these aren't linked to from anywhere by default, should we mention them in the documentation somewhere so that people know they exist?

**Comment by hodgestar** [comment]
Separate thought: Is it worthwhile adding a markup extension that allows generic access to something like Django's `url_for` in templates?

**Comment by drnlm** [comment]
A markdown extension supporting url_for sounds like a great addition.

File: wafer/talks/templates/wafer.talks/talk_tracks.html [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{% extends "wafer/base.html" %}
+{% load i18n %}
+{% block title %}{% trans "Tracks" %} - {{ WAFER_CONFERENCE_NAME }}{% endblock %}
+{% block content %}
+<section class="wafer wafer-talk-tracks">
+  <h1>{% trans 'Tracks' %}</h1>
+    <ul class="track-list">
+    {% for track in object_list %}
+       <li>{{ track.name }} &mdash; {{ track.description }}</li>
```
Comment (stefanor) [inline wafer/talks/templates/wafer.talks/talk_tracks.html]: Did you consider using HTML definition lists?

Comment (drnlm) [reply wafer/talks/templates/wafer.talks/talk_tracks.html]: No - it's one of those features of HTML I always forget about.

**Pull request merged**